protos/sources/vega/checkpoint/v1/checkpoint.proto

```diff
@@ -224,3 +225,64 @@ message AssetAction {
   bool erc20_bridge_stopped = 11;
   bool erc20_bridge_resumed = 12;
 }
+
+message LPBalance {
+  // Party ID of the LP
+  string party_id = 1;
+  // The actual balance of the bond account.
+  string bond_balance = 2;
+}
+
+message ELSShare {
+  // the party if of the LP.
+  string party_id = 1;
+  // their share as tracked by the ELS component (nominal amount).
+  string share = 2;
+  // the amount committed (the total commitment made - does not necessarily match the LP balance or share)
+  string supplied_stake = 3;
+  // the virtual stake.
+  string virtual_stake = 4;
+  // the average valuation
+  string avg = 5;
```

Comment:
```suggestion
  // Party ID of the liquidity provider.
  string party_id = 1;
  // Liquidity provider's share as tracked by the ELS component; nominal amount.
  string share = 2;
  // Total amount committed. This does not necessarily match the LP balance or share
  string supplied_stake = 3;
  // Virtual stake.
  string virtual_stake = 4;
  // Average valuation.
  string avg = 5;
````

Reply:
Can you add a little more info here about average valuation and virtual stake?

Reply:
:D a little more information... These are internal values part of the equity like share calculations, it's all quite complex to document. For example: virtual stake is dependent on the market growth, whether you've increased or decreased your commitment, and how long you've held that stake, as outlined in the specs here: https://github.com/vegaprotocol/specs/blob/734c728dde0b4374d2dd2e7dc64ce6a8e11eb821/protocol/0042-LIQF-setting_fees_and_rewarding_lps.md?plain=1#L68

> i virtual stake <- max(LP i physical stake, (1 + r) x (LP i virtual stake)).

Where `r` is market growth over a given market window